src/test/resources/bkg.v1/BookingRequestsSchema.json

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
Shouldn't this file be named `BookingSummarySchema.json` since it's specifically a summary?

Reply:
Yes, I agree.